deps.go

```diff
@@ -0,0 +1,117 @@
+package main
```

Comment:
Is this a leftover ? Or a new tool/script ? If it's a new tool/script we can perhaps in the future get a directory for it ?

Reply:
I would like to see it as a new tool. It is surely incomplete, but already useful.
And yes, we should have a durectory to contain it.
It is very helpful to check not needed headers.